modules/carto/src/api/maps-v3-client.ts

```diff
@@ -89,7 +89,12 @@ async function requestJson<T = unknown>({
   errorContext
 }: RequestParams): Promise<T> {
   const response = await request({method, url, headers, accessToken, body, errorContext});
-  const json = await response.json();
+  let json;
+  try {
+    json = await response.json();
+  } catch (error) {
+    json = {error: ''};
```

Comment:
is it temporary? if yes, please mark as TODO, so proper error handling is reverted later